autoPyTorch/evaluation/abstract_evaluator.py

```diff
@@ -374,7 +375,7 @@ def _loss(self, y_true: np.ndarray, y_hat: np.ndarray) -> Dict[str, float]:
                 For logloss for example: optimum(0) - (-1 * actual score)
         """
 
-        if not isinstance(self.configuration, Configuration):
+        if isinstance(self.configuration, int):
             return {self.metric.name: 1.0}
```

Comment:
I am sorry, I know this is not related to this PR, but can you update this to be self.metric-> worst possible result instead of 1.0?

Reply:
Well actually a loss....

Reply:
Definitely, I can do this